app/models/project.js

```diff
@@ -53,4 +53,4 @@ export default class ProjectModel extends Model.extend({}) {
       data: turfBuffer(data, 0.113636, { units: 'miles' })
     };
   }
-}
+}
```

Comment:
I would undo this change just because the best practice is to have a newline at the end of the file